### PR TITLE
Build deb package

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -18,4 +18,51 @@ jobs:
...
+          ./debian/prepare-deb-build.sh
+      - uses: jiro4989/build-deb-action@v3
+        id: deb
+        with:
+          package: ya-installer-resources
+          package_root: .debpkg
+          maintainer: "Golem Factory <contact@golem.network>"
+          version: ${{ github.ref }}
+          arch: 'amd64'
+          depends: 'golem-provider'
```
Comment (pwalski) [inline .github/workflows/release.yml]: AFAIK `ya-provider pre-install` appeared in `0.13`
```suggestion
          depends: 'golem-provider (>= 0.13.0)'
```

File: debian/postinst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+resources_dir="${RESOURCES_UNPACK_DIR:-/tmp/ya-installer/}"
+
+ya-provider pre-install
+golemsp manifest-bundle add "${resources_dir}"
+
+rm -r "${resources_dir}"
```
Comment (pwalski) [inline debian/postinst]: To avoid prompt
```suggestion
rm -rf "${resources_dir}"
```

**Review by pwalski — CHANGES_REQUESTED** [review]
This `golem-provider` version requirement is a must IMO

File: debian/postinst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+resources_dir="${RESOURCES_UNPACK_DIR:-/tmp/ya-installer/}"
```
Comment (pwalski) [inline debian/postinst:5]: Is there a case where user might want to use different directory?
Do you want to let user to do `export RESOURCES_UNPACK_DIR=$HOME` and then run `rm -r` on it?
Maybe it should create `${RESOURCES_UNPACK_DIR}/$VERSION` and then delete it? (no idea how to access `VERSION`)

**Review by pwalski — APPROVED** [review]

File: debian/postinst (hunk 3 above)
Comment (nieznanysprawiciel) [reply debian/postinst:5]: it's good point, it is dangerous.
There is a case, but not in related to debs. I would like to use single script in all places (`ya-installer`) and there you would need to provider directory

Comment (pwalski) [reply debian/postinst:5]: If this env var is meant only for some internal use then it is fine.

File: debian/prepare-deb-build.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+tmp_dir=.debpkg/tmp/ya-installer
+
+mkdir -p ${tmp_dir}
+mkdir -p .debpkg/DEBIAN
+
+cp -r certs/ ${tmp_dir}/certs
+cp -r whitelist/ ${tmp_dir}/whitelist
```
Comment (pwalski) [inline debian/prepare-deb-build.sh:9]:
```suggestion
rm -rf ${tmp_dir}/whitelist && cp -r whitelist/ ${tmp_dir}/whitelist
```

**Pull request merged**